### PR TITLE
login

### PR DESCRIPTION
**Pull request by brunomichetti**

### CODE REVIEW

File: users/serializers.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from rest_framework import serializers
+from django.contrib.auth.models import User
```
Comment (fedeagripa) [inline users/serializers.py]: remember to leave empty line between

File: users/tests.py [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
 
-# Create your tests here.
+class LoginTestCase(TestCase):
+    
+    #define setUpClass to initial configuration    
+    @classmethod
+    def setUpClass(self):
+        super().setUpClass()
+        self.client = Client()
+        # command to load the fixture
```
Comment (fedeagripa) [inline users/tests.py]: remove comment please

File: users/tests.py [hunk 3]
```diff
@@ -1,3 +1,24 @@
-from django.test import TestCase
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
+from django.core.management import call_command
 
-# Create your tests here.
+class LoginTestCase(TestCase):
+    
+    #define setUpClass to initial configuration
```
Comment (fedeagripa) [inline users/tests.py]: same here

File: target/settings.py [hunk 4]
```diff
@@ -37,6 +37,10 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'users',
```
Comment (kaozdl) [inline target/settings.py]: As a standard try to put your apps below the django apps and thrid party apps that you install

File: users/urls.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+from django.conf.urls import url
+from django.urls import path
+
+app_name = 'users'
```
Comment (kaozdl) [inline users/urls.py]: This should not be in apps.py?
Why don´t you have any urlpatterns?

**Comment by kaozdl** [comment]
You have some .pyc binary files tracked in the repo. 
maybe they were added before the gitignore file.

**Comment by kaozdl** [comment]
Excelent.
For the users it wont be necessary, but maybe for the targets you can copy the rails implementation of the test suite. Django now runs all the test in every file that starts with a test_* and inherits from APITestCase

**Pull request merged**